std/file.d

```diff
@@ -2287,21 +2287,21 @@ if (isConvertibleToString!R)
 // Same as mkdir but ignores "already exists" errors.
 // Returns: "true" if the directory was created,
 //   "false" if it already existed.
-private bool ensureDirExists(in char[] pathname)
+private bool ensureDirExists(in char[] pathname) @safe
 {
     import std.exception : enforce;
 
     version(Windows)
     {
-        if (CreateDirectoryW(pathname.tempCStringW(), null))
+        if (() @trusted { return CreateDirectoryW(pathname.tempCStringW(), null); }())
```

Comment:
> Does by chance anyone know whether this overhead is negligible after optimization?

Depends on the in-liner. Chances are DMD won't in-line this, but I wouldn't change this, as optimizing for DMD's in-liner is a deep rabbit hole we don't want to go down.